lib/rooms/room.ex

```diff
@@ -125,6 +126,7 @@ defmodule SignalTower.Room do
         send_peer_left(next_members, peer_id)
         {:ok, {room_id, next_members}}
       else
+        SignalTower.PrometheusStats.room_closed()
```

Comment:
This must also work on l. 61, can you please check it again?

Reply:
I was wrong, leave is also (and actually most importantly) called in l.108 when the websocket process goes away, so this is the (only) correct spot to put this - sorry for the confusion.

Reply:
Np